test/suite/test_cursor13.py

```diff
@@ -527,3 +527,22 @@ def test_cursor_sweep(self):
         # by approximately the number of swept cursors, but it's less
         # predictable.
         self.assertGreater(end_stats[1] - begin_stats[1], 0)
+
+class test_cursor13_dup(test_cursor13_base):
+    def test_dup(self):
+        self.cursor_stats_init()
+        uri = 'table:test_cursor13_dup'
+        self.session.create(uri, 'key_format=S,value_format=S')
+        cursor = self.session.open_cursor(uri)
+        cursor['A'] = 'B'
+        cursor.close()
+
+        c1 = self.session.open_cursor(uri, None)
+        c1.next()
```

Comment:
It took a moment to understand the `next` call. Add a short comment that we need to position the cursor on the data before dup'ing it.